LabExT/Movement/MoverNew.py

```diff
@@ -615,54 +631,37 @@ def move_relative(
             with calibration.perform_in_system(CoordinateSystem.CHIP):
                 calibration.move_relative(requested_target, wait_for_stopping)
 
-    @contextmanager
-    def lift_stages(self):
-        """
-        Lifts the stages to the configured value before the block is executed and lowers the stages again after the block.
-
-        Performs movement in chip coordinates.
-        Performs NO safe movement (i.e no Path planning)
-        """
-
-        def _lift_lower_stages(lift):
-            for calibration in self.calibrations.values():
-                with calibration.perform_in_system(CoordinateSystem.CHIP):
-                    calibration.move_absolute(
-                        calibration.get_position() + ChipCoordinate(z=lift))
-
-        _lift_lower_stages(self.z_lift)
-        try:
-            yield
-        finally:
-            _lift_lower_stages(-self.z_lift)
-
     @assert_connected_stages
     def move_to_device(self, chip: Type[Chip], device: Type[Device]):
         """
         Moves stages to device.
 
+        Moves stages absolute to coordinate with path planning and lifted stages.
+
         Parameters
         ----------
         chip: Chip
             Instance of a imported chip.
         device: Device
             Device to which the stages should move.
-
-        Raises
-        ------
-        MoverError
-            If no device was found for the given ID.
         """
         movement_commands = {}
-        if self.input_calibration:
-            movement_commands[self.input_calibration.orientation] = device.input_coordinate + \
-                ChipCoordinate(z=self.z_lift)
-        if self.output_calibration:
-            movement_commands[self.output_calibration.orientation] = device.output_coordinate + \
-                ChipCoordinate(z=self.z_lift)
 
-        with self.lift_stages():
-            self.move_absolute(movement_commands, chip=chip)
+        input_orientation = self._port_by_orientation.inverse.get(
+            DevicePort.INPUT)
+        output_orientation = self._port_by_orientation.inverse.get(
+            DevicePort.OUTPUT)
+
+        if input_orientation:
+            movement_commands[input_orientation] = device.input_coordinate
+
+        if output_orientation:
+            movement_commands[output_orientation] = device.output_coordinate
+
+        self.move_absolute(
+            movement_commands,
+            chip=chip,
+            with_lifted_stages=True)
```

Comment:
`move_to_device` is now very simple: It looks at which orientation an input or output stage was configured. Accordingly, the movement commands (mapping from orientation to target chip coordinate) are constructed and passed to the move absolute.